### ISSUE TITLE
"java.lang.IllegalArgumentException: Unsupported enum value type: java.lang.Byte" occurs when scanning

### ISSUE THREAD

**Issue by ahamana** [lukehutch/fast-classpath-scanner#208]
Many thanks for your great library.

I have a question.

I encountered the following exception when the fastclasspathscanner scanned OSS libraries of my web application.
```
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- Found class org.hibernate.validator.internal.constraintvalidators.bv.money.DecimalMinValidatorForMonetaryAmount
2018-06-25T20:28:26.825+0900	FastClasspathScanner	------ Superclass: java.lang.Object
2018-06-25T20:28:26.825+0900	FastClasspathScanner	------ Interfaces: javax.validation.ConstraintValidator
2018-06-25T20:28:26.825+0900	FastClasspathScanner	------ Class type signature: public synchronized class org.hibernate.validator.internal.constraintvalidators.bv.money.DecimalMinValidatorForMonetaryAmount implements javax.validation.ConstraintValidator<javax.validation.constraints.DecimalMin, javax.money.MonetaryAmount>
2018-06-25T20:28:26.825+0900	FastClasspathScanner	-- Exception in worker thread
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- java.lang.IllegalArgumentException: Unsupported enum value type: java.lang.Byte
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.AnnotationInfo$AnnotationParamValueWrapper.<init>(AnnotationInfo.java:128)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.AnnotationInfo$AnnotationParamValue.<init>(AnnotationInfo.java:208)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.ClassfileBinaryParser.readClassInfoFromClassfileHeader(ClassfileBinaryParser.java:913)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.ClasspathElement.parseClassfiles(ClasspathElement.java:297)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.Scanner$6.processWorkUnit(Scanner.java:446)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.scanner.Scanner$6.processWorkUnit(Scanner.java:440)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.utils.WorkQueue.runWorkLoop(WorkQueue.java:187)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.utils.WorkQueue$1.call(WorkQueue.java:147)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at io.github.lukehutch.fastclasspathscanner.utils.WorkQueue$1.call(WorkQueue.java:144)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-06-25T20:28:26.825+0900	FastClasspathScanner	---- 	at java.lang.Thread.run(Thread.java:748)
```

For instance, what kind of causes are there?

If you don't know, please ignore this question and close this issue.

**Comment by johnou** [comment]
@ahamana please provide the jar or class it blew up with.

**Comment by lukehutch** [comment]
@ahamana Thanks for this report. There was a copy/paste error in a recent code change. Fixed in 3.1.6.

**Comment by ahamana** [comment]
@lukehutch Many thanks for fixing the issue quickly.
